core/src/main/java/org/apache/carbondata/core/util/CleanFilesUtil.java

```diff
@@ -146,8 +146,8 @@ public static void cleanStaleSegmentsForPartitionTable(CarbonTable carbonTable)
    * in the metadata folder and is not present in the table status file is considered as a
    * stale segment. Only comparing from tablestatus file, not checking tablestatus.history file
    */
-  private static void getStaleSegmentFiles(CarbonTable carbonTable, List<String> staleSegmentFiles,
-      List<String> redundantSegmentFile) {
+  private static void collectStaleSegmentFiles(CarbonTable carbonTable,
```

Comment:
I suggest `getStaleSegmentFiles` is fine, but make a list inside the function and send as a return value.  Passing as an argument makes sense only when the list has some pre-entries or used in recursion.

Reply:
this method need return two lists, so it passes two lists in parameters

Reply:
ok, doesn't make much difference